### ISSUE TITLE
NSInvalidArgumentException +[UIImage systemImageNamed:] 13.0.0 <= iOS version < 13.4.0

### ISSUE THREAD

**Issue by Tulakshana** [xmartlabs/Eureka#2050]
pod 'Eureka', '~> 5.2.1'
> - Make sure you are using Eureka [latest release](https://github.com/xmartlabs/Eureka/releases) or master branch version. 
> - Make sure your Xcode version is the latest stable one.

Issue not previously reported
> - Check if the issue was [already reported or fixed](https://github.com/xmartlabs/Eureka/issues?utf8=%E2%9C%93&q=is%3Aissue). We add labels to each issue in order to easily find related issues. If you found a match add a brief comment "I have the same problem" or "+1".

- Environment: 
Eureka 5.2.1
Xcode 11.4.1
13.0.0 <= iOS version < 13.4.0

- Stack trace from Crashlytics
<img width="1225" alt="Screen Shot 2020-06-09 at 9 48 43 AM" src="https://user-images.githubusercontent.com/1791770/84182552-702ed380-aa3f-11ea-88ad-760ea763aeea.png">

- Eureka form/section/row code
<img width="997" alt="Screen Shot 2020-06-09 at 11 05 40 AM" src="https://user-images.githubusercontent.com/1791770/84183716-2e069180-aa41-11ea-8b26-c02424ee58d3.png">

My team and I concluded that Apple is at fault here for the following reasons,

- We didn't see it happening in iOS versions > 13.4.0 so Apple might have fixed the issue
- This was not the only selector/property iOS failed to recognize in 13.0.0 <= iOS version < 13.4.0. We observed that some other selectors and properties introduced in iOS 13 were also throwing the same error and result in a crash. e.g. UISearchTextField property of UISearchBar also caused the same problem in 13.0.0 <= iOS version < 13.4.0.

I'm reporting this here thinking that it will help someone.

**Comment by mats-claassen** [comment]
Thanks for reporting this and for the investigation too.

**Comment by nicolantean** [comment]
Hey @Tulakshana I've tried to replicate your issue but I couldn't. I'm using Xcode 11.5 and an iOS 13.1 device. Can you send an example code where the issue happens?

**Comment by Tulakshana** [comment]
We were also not able reproduce it ourselves. Only evidence we have is the crash report.

**Comment by nicolantean** [comment]
Does that crash happen to all your users that have iOS devices with versions between 13.0 and 13.4 or only some specific ones?

**Comment by Tulakshana** [comment]
Good question. Looks like it is a subset.

**Comment by crgt5252** [comment]
We are seeing this issue on > iOS 14.  Cannot repro, just seeing it in crash logs. 

![Uploading Screen Shot 2021-03-19 at 3.12.54 PM.png…]()

**Comment by Tulakshana** [comment]
Yes, we also ended up commenting out code specific iOS 13 and above.

**Comment by hongmh** [comment]
In my case, it happens with Jailbroken devices. 

<img width="434" alt="スクリーンショット 2021-04-05 9 42 43" src="https://user-images.githubusercontent.com/17564411/113526047-4872b280-95f3-11eb-8940-a912f4c7a6c0.png">
